src/mdFormBuilder.html

```diff
@@ -20,9 +20,11 @@
 		<div ng-if="FormBuilder.displayType === null">
 	    <div ng-repeat="section in FormBuilder.sections" flex="{{ section.flex }}">
 	      <h3 style="padding: 15px 25px; margin: 0px; border-bottom: 1px dashed #ccc" ng-if="section.displayName">{{ section.displayName }}</h3>
-
 	    	<div ng-repeat="row in section.rows" layout-padding>
 					<h4 style="margin: 0px 10px; font-size: 14px;" ng-if="row.title">{{ row.title }}</h4>
+					<div ng-if="row.enableRowRemoval" layout="row" layout-align="end end">
```

Comment:
I think we can remove th ng-if and place it in the md-icon directly.
Then we need to add additional styles to this element:
position: absolute;
top: 0px;
right: 0px;

This floats the element right.